litellm/llms/prompt_templates/factory.py

```diff
@@ -1460,7 +1384,41 @@ def anthropic_messages_pt(  # noqa: PLR0915
 
             msg_i += 1
 
-        if user_content:
+        if user_content and user_message_types_block["role"] in {"tool", "function"}:
+            if user_message_types_block["role"] == "tool":
```

Comment:
This part is copied from `convert_to_anthropic_tool_result` without any modification